db/bigtable.go

```diff
@@ -836,6 +828,87 @@ func (bigtable *Bigtable) GetValidatorAttestationHistory(validators []uint64, st
 	return res, nil
 }
 
+func (bigtable *Bigtable) GetValidatorMissedAttestationHistory(validators []uint64, startEpoch uint64, endEpoch uint64) (map[uint64]map[uint64]bool, error) {
```

Comment:
This function duplicates a lot of code from `GetValidatorAttestationHistory` in order to be faster. Have you measured the gain though? I fell like it might not be worth the effort.

Reply:
It was not only a performance but also a memory issue where the GetValidatorAttestationHistory used way to much memory. Memory went above 5gig with the old way, now its max 1 gig.

Reply:
Ah, I see.